frontend/src/reducers/queries.ts

```diff
@@ -0,0 +1,216 @@
+import { IQuery } from '../common/models/apps/querys';
+import { Routine } from 'redux-saga-routines';
+import {
+  duplicateSelectQueryRoutine,
+  fetchQueryRoutine,
+  openQueryRoutine,
+  setNewCodeRoutine,
+  setNewConfirmRoutine,
+  setNewNameQueryRoutine,
+  setNewRunRoutine,
+  setSelectQueryRoutine, setSuccessTriggersRoutine,
+  setUnSuccessTriggersRoutine,
+  setWaiterQueryRoutine
+} from '../scenes/constructor/routines';
+import { ITrigger } from '../common/models/query/ITrigger';
+
+export interface IQueryState {
+  queriesApp: Array<IQuery>;
+  selectQuery:{
+    selectQueryId: string,
+    selectQueryName: string,
+    selectQueryCode: string,
+    selectQueryTriggers:Array<ITrigger>|[],
+    runAutomatically: boolean,
+    showConfirm: boolean
```

Comment:
move it to models, then reducer will be cleaner, now there is too much information